app/gui/src/main/scala/org/bitcoins/gui/dlc/DLCPane.scala

```diff
@@ -90,38 +90,17 @@ class DLCPane(glassPane: VBox) {
     }
   }
 
-  private val initCloseButton = new Button {
-    text = "Init Close"
-    onAction = new EventHandler[ActionEvent] {
-      override def handle(event: ActionEvent): Unit = model.onInitClose()
-    }
-  }
-
-  private val acceptCloseButton = new Button {
-    text = "Accept Close"
-    onAction = new EventHandler[ActionEvent] {
-      override def handle(event: ActionEvent): Unit = model.onAcceptClose()
-    }
-  }
-
   private val refundButton = new Button {
     text = "Refund"
     onAction = new EventHandler[ActionEvent] {
       override def handle(event: ActionEvent): Unit = model.onRefund()
     }
   }
 
-  private val forceCloseButton = new Button {
-    text = "Force Close"
-    onAction = new EventHandler[ActionEvent] {
-      override def handle(event: ActionEvent): Unit = model.onForceClose()
-    }
-  }
-
-  private val punishButton = new Button {
-    text = "Punish"
+  private val closeButton = new Button {
+    text = "Close"
```

Comment:
Nit: I personally prefer `execute` to `close` here and in related places, your call tho